app/screens/ChildSetupList.tsx

```diff
@@ -88,6 +90,18 @@ const ChildSetupList = ({ navigation }: Props): any => {
     React.useCallback(() => {
       getAllChildren(dispatch, childAge, 0);
       getAllConfigData(dispatch);
+      async function requestNotificationPermission(): Promise<any>{
+        const status= await requestNotifications([]);
+        console.log(status,"..status..")
+      }
+      setTimeout(()=>{
+        if(Platform.OS=="android"){
+          requestNotificationPermission();
+        }
+        else{
+          PushNotification.requestPermissions(); 
+        }  
+      },100)
```

Comment:
Should we make this a reusable method if it is being implemented more then once?

Reply:
function made as common